plugin/aws/ec2Instance.go

```diff
@@ -43,8 +44,17 @@ const (
 )
 
 func newEC2Instance(ctx context.Context, inst *ec2Client.Instance, session *session.Session, client *ec2Client.EC2) *ec2Instance {
+	id := awsSDK.StringValue(inst.InstanceId)
+	name := id
+	for _, tag := range inst.Tags {
+		if awsSDK.StringValue(tag.Key) == "Name" {
```

Comment:
This seems to suggest that our present use of `e.Name()` should really be `e.ID()` (where ID is the entry's ID w.r.t to the plugin API) and `e.Name` is `e.CName` (or what's displayed in the shell). Both `CName` and `ID` are configurable by plugin authors.

I don't think we currently need to change anything in the plugin architecture, but this is an interesting thing to keep in mind to see if other plugins will adopt similar conventions.

Reply:
Also, a comment explaining what's in the `Name` tag would be nice.